src/test/java/edu/phema/elm_to_omop/phenotype/FilePhenotypeTest.java

```diff
@@ -30,7 +35,17 @@
     public void setup() throws Exception {
         MockitoAnnotations.initMocks(this);
 
-        lenient().when(omopRepository.getConceptMetadata("1.2.3.4")).thenReturn(new Concept());
+        // mock for method vocabularySearch
+        when(omopRepository.vocabularySearch(anyString(), anyString())).thenAnswer(new Answer<ArrayList<Concept>>() {
```

Comment:
I think this approach is better than the WireMock approach I've been using. We should migrate to using this (not in this PR though).